site/content/docs/latest/howto/access-control.md

```diff
@@ -1,30 +1,52 @@
 # Access Control in Kubeapps
 
-Before you configure the Kubernetes Role-Based-Access-Control for your authenticated users to access Kubeapps you will first need to establish how users will authenticate with the Kubernetes clusters on which Kubeapps operates.
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [User Authentication in Kubeapps](#user-authentication-in-kubeapps)
+   1. [OpenID Connect authentication](#openid-connect-authentication)
+   2. [Service Account authentication](#service-account-authentication)
+3. [Assigning Kubeapps User Roles](#assigning-kubeapps-user-roles)
+   1. [Applications](#applications)
+      1. [Read access to Applications within a namespace](#read-access-to-applications-within-a-namespace) 2.[Write access to Applications within a namespace](#write-access-to-applications-within-a-namespace)
+   2. [Package Repositories](#package-repositories)
+   3. [Assigning roles across multiple namespaces](#assigning-roles-across-multiple-namespaces)
+   4. [Using a cluster-admin user (not recommended)](#using-a-cluster-admin-user-not-recommended)
+
+## Introduction
+
+[Kubeapps](https://kubeapps.dev/) provides a cloud native solution to browse, deploy and manage the lifecycle of applications on a Kubernetes cluster. It is a one-time install that gives you a number of important benefits, including the ability to:
+
+- browse and deploy packaged applications from public or private repositories
+- customize deployments through an intuitive user interface
+- upgrade, manage and delete the applications that are deployed in your Kubernetes cluster
+- expose an API to manage your package repositories and your applications
+
+This guide explains in detail how to **configure the Kubernetes Role-Based-Access-Control (RBAC) for your authenticated users to access Kubeapps**. Before configuring the RBAC, you need to establish how users authenticate with the Kubernetes clusters on which Kubeapps operates.
 
 ## User Authentication in Kubeapps
 
 By design, Kubeapps does not include a separate authentication layer but rather relies on the [supported mechanisms provided with Kubernetes itself](https://kubernetes.io/docs/reference/access-authn-authz/authentication/).
 
-Each request to Kubeapps requires a token trusted by the Kubernetes API token in order to make
-requests to the Kubernetes API server as the user. This ensures that a certain
+Each request to Kubeapps requires a token trusted by the Kubernetes API token in order to make requests to the Kubernetes API server as the user. This ensures that a certain
 user of Kubeapps is only permitted to view and manage applications to which they
 have access (for example, within a specific namespace). If a user does not
```

Comment:
`a certain user`...`to which they have access`. Not sure, but does the subject match?

Reply:
I think it is to avoid "he/she", so using "they".

Reply:
Exactly, that's right, see https://en.wikipedia.org/wiki/Singular_they